resources/views/components/cart-nav.blade.php

```diff
@@ -0,0 +1,85 @@
+@if (Route::has('login'))
+    @auth
+         <section class="bg-white border-b border-solid border-[#E0E0E0]">
+    <div class="container mx-auto">
+      <div class="max-w-screen-xl mx-auto">
+        <div class="flex items-center justify-between">
+          <div class="text-base font-medium text-center text-[#000]">
+            <ul class="flex flex-wrap -mb-px">
+              <li class="me-2">
+                <a href="{{ url('/cart') }}"
```

Comment:
@kams2391 Assign name to the route and use it here
i.e {{ route('cart') }}

Reply:
Do this for all link

Reply:
Done. Added name for all links.